UKHomeOffice/nmsw-ui
# NMSW-470 p3 Refactor declarationId to come from params not state - Crew details

# Ticket

NMSW-470

----

## AC

see https://github.com/UKHomeOffice/nmsw-ui/pull/269

----

## To test
- create a voyage
- add a general declaration fal 1
- click check for errors
- > you should see the success page
- click save and continue
- > you should see the task details page
- click the crew link
- > you should be taken to the crew page with the declarationId in the url
- select a bad file
- click check fr errors
- > you should see the error page
- click re-upload file
- select a good file
- click check for errors
- > you should see the success page
- click save and continue
- > you should return to the task details page

----

## Notes
The following pages will not work as they've not yet been updated

- passenger
- supporting docs
- CYA
- delete
